packages/wallet/test/SingleAddressWallet/load.test.ts

```diff
@@ -93,6 +93,8 @@ const assertWalletProperties = async (
   expect(await firstValueFrom(wallet.assets$)).toEqual(new Map([[AssetId.TSLA, mocks.asset]]));
   // timeSettings$
   expect(await firstValueFrom(wallet.timeSettings$)).toEqual(testnetTimeSettings);
+  // inputAddressResolver
```

Comment:
Maybe we can remove this comment as well as `// timeSettings$` ?

Reply:
This test function use comments to group what's being tested. A little odd for a single line, but helps for some, e.g.
```ts
  // addresses$
  const addresses = await firstValueFrom(wallet.addresses$);
  expect(addresses[0].address).toEqual(address);
  expect(addresses[0].rewardAccount).toEqual(rewardAccount);
  // assets$
  ...
```